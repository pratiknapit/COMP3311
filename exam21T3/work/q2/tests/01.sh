psql property -c 'select * from q2;'
