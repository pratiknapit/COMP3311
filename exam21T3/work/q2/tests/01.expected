     suburb     |   ptype   | nprops 
----------------+-----------+--------
 Alexandria     | Apartment |     42
 Bondi Junction | Apartment |     41
 Coogee         | Apartment |     47
 Kingsford      | Apartment |     43
 Alexandria     | House     |     34
 Bondi Junction | House     |     38
 Coogee         | House     |     38
 Kingsford      | House     |     36
 Alexandria     | Townhouse |      9
 Bondi Junction | Townhouse |      9
 Coogee         | Townhouse |      6
 Kingsford      | Townhouse |      9
(12 rows)
